src/odemis/acq/move.py

```diff
@@ -179,26 +180,34 @@ def cryoLoadSample(target):
     f._task_lock = threading.Lock()
     f._running_subf = model.InstantaneousFuture()
     # Run in separate thread
-    executeAsyncTask(f, _doCryoLoadSample, args=(f, stage, focus, target))
+    executeAsyncTask(f, _doCryoSwitchSamplePosition, args=(f, stage, focus, target))
     return f
 
 
-def _doCryoLoadSample(future, stage, focus, target):
+def _doCryoSwitchSamplePosition(future, stage, focus, target):
     """
-    Do the actual switching procedure for the Cryo sample stage between loading and imaging
+    Do the actual switching procedure for the Cryo sample stage between loading, imaging and coating positions
     :param future: cancellable future of the move
     :param stage: sample stage that's being controlled
     :param focus: focus for optical lens
-    :param target: target position either one of the constants LOADING or IMAGING
+    :param target: target position either one of the constants LOADING, IMAGING and COATING
     """
     try:
         stage_md = stage.getMetadata()
         focus_md = focus.getMetadata()
         stage_active = stage_md[model.MD_FAV_POS_ACTIVE]
         stage_deactive = stage_md[model.MD_FAV_POS_DEACTIVE]
         stage_coating = stage_md[model.MD_FAV_POS_COATING]
-        stage_active_range = stage_md[model.MD_POS_ACTIVE_RANGE]
         focus_deactive = focus_md[model.MD_FAV_POS_DEACTIVE]
+        # Some sanity checks for rotation axes
+        for rotation_axis in {'rx', 'rz'}:
+            if rotation_axis in stage_active:
+                logging.warning("Imaging position should not contain a value for %s angle." % rotation_axis)
```

Comment:
Changed this to check for both rotation angels, because `rx` is missing in the stage `FAV_POS_ACTIVE` on enzel-tudelft yaml files

Reply:
Let's just keep the whole code simple. Every metadata has to have every axis. If an axis is missing, it raises an error.